apps/glusterfs/volume_durability_test.go

```diff
@@ -77,41 +77,41 @@ func TestNoneDurability(t *testing.T) {
 	r := &NoneDurability{}
 	r.SetDurability()
 
-	gen := r.BrickSizeGenerator(100 * GB)
+	gen := r.BrickSizeGenerator(50 * GB)
 
 	// Gen 1
 	sets, brick_size, err := gen()
 	tests.Assert(t, err == nil)
 	tests.Assert(t, sets == 1)
-	tests.Assert(t, brick_size == 100*GB)
+	tests.Assert(t, brick_size == 50*GB)
 	tests.Assert(t, 1 == r.BricksInSet())
 
 	// Gen 2
 	sets, brick_size, err = gen()
 	tests.Assert(t, err == nil)
 	tests.Assert(t, sets == 2)
-	tests.Assert(t, brick_size == 50*GB)
+	tests.Assert(t, brick_size == 25*GB)
 	tests.Assert(t, 1 == r.BricksInSet())
 
 	// Gen 3
 	sets, brick_size, err = gen()
 	tests.Assert(t, err == nil)
 	tests.Assert(t, sets == 4)
-	tests.Assert(t, brick_size == 25*GB)
+	tests.Assert(t, brick_size == 12800*1024)
 	tests.Assert(t, 1 == r.BricksInSet())
 
 	// Gen 4
 	sets, brick_size, err = gen()
 	tests.Assert(t, err == nil)
 	tests.Assert(t, sets == 8)
-	tests.Assert(t, brick_size == 12800*1024)
+	tests.Assert(t, brick_size == 6400*1024)
 	tests.Assert(t, 1 == r.BricksInSet())
 
 	// Gen 5
 	sets, brick_size, err = gen()
 	tests.Assert(t, err == nil)
 	tests.Assert(t, sets == 16)
-	tests.Assert(t, brick_size == 6400*1024)
+	tests.Assert(t, brick_size == 3200*1024)
 	tests.Assert(t, 1 == r.BricksInSet())
 
 	// Gen 6
```

Comment:
Well, then we should do :-)
Also go additional steps further down, as needed.

Alternatively, we could also change the config that we run the test on to have 4 GB BrickMinSize ... but I don't think we should do that.